app/controllers/alerts_list_controller.rb

```diff
@@ -16,6 +17,15 @@ def index
     redirect_to :action => 'show'
   end
 
+  def class_icons
+    res = {}
+    icons.each do |k, _val|
+      file_name = "100/#{k.to_s.underscore}.png"
```

Comment:
(Don't worry too much about having the method there twice (once class once instance), we were talking about doing something like defaulting to the class method unless the instance method needs to exist because the icon depends on some state, but for now... shrug :).)